tests/sync/test_smb.py

```diff
@@ -1,17 +1,26 @@
+import io
 import pathlib
 import socket
+import logging
 
 import pytest
 import attr
 import keyring
 from smb.SMBConnection import SMBConnection
-from smb.smb_structs import OperationFailure
+from smb.smb_structs import OperationFailure, ProtocolError
 
 from kitovu.sync import syncing
 from kitovu import utils
 from kitovu.sync.plugin import smb
 
 
+@attr.s
+class FakeStatResult:
```

Comment:
Copied from `test_moodle.py`, but I think it doesn't make much sense to move this to a helper module for this little code.